pontoon/base/static/js/translate.js

```diff
@@ -271,6 +282,24 @@ var Pontoon = (function (my) {
 
 
     /*
+     * Open batch editor
+     */
+    openBatchEditor: function () {
```

Comment:
Idea for the next refactorings: I guess we could create an separate object called an Editor and add proper methods like setSelectedCount. At some point it's a really hard to figure what selectors are really meaning for the application.
However, that's not something we should do in this pull request.